telegram/ext/callbackqueryhandler.py

```diff
@@ -163,6 +164,8 @@ def check_update(self, update: object) -> Optional[Union[bool, object]]:
                     match = re.match(self.pattern, update.callback_query.data)
                     if match:
                         return match
+                else:
+                    return None
```

Comment:
This isn't necessary to up the coverage ;)

Reply:
and here I thought thats the whole point? I am confused now.

Reply:
no, codecov only tells us to have a test for the if-clause giving "true" and a test for the if-clause giving "false". It doesn't tell us that we need to have an else-clause 😉

Reply:
ohhhhh. Thats what its doing. And it knows where the line is tested in pytest. Damn thats smart coding

Reply:
> And it knows where the line is tested in pytest

No, it doesn't. Running pytest with the right arguments (and additional dependencies) just generates a report about which lines are covered and codecov can read that

Reply:
Dependencies: 

https://github.com/python-telegram-bot/python-telegram-bot/blob/9737b1d3c78bdb8d79d726e27a2b37f840fe3f55/.github/workflows/test.yml#L31

arguments:

https://github.com/python-telegram-bot/python-telegram-bot/blob/9737b1d3c78bdb8d79d726e27a2b37f840fe3f55/.github/workflows/test.yml#L49

uploading report:

https://github.com/python-telegram-bot/python-telegram-bot/blob/9737b1d3c78bdb8d79d726e27a2b37f840fe3f55/.github/workflows/test.yml#L62

see e.g. https://github.com/python-telegram-bot/python-telegram-bot/runs/1937444759#step:7:38